lesson_1.1/src/App.js

```diff
@@ -1,12 +1,14 @@
-import React from "react";
+import {react, useState} from "react"
```

Comment:
import statement written incorrectly. You should have it written as:

import React, { useState } from 'react';

since useState is a named export from the react library so it is in curly braces. React is the default export from the react library and not a specific named export like useState.